doc/nrf/device_guides/working_with_nrf/nrf53/qspi_xip_guide.rst

```diff
@@ -273,6 +273,11 @@ Similarly, it is possible to relocate certain libraries, for example:
    zephyr_code_relocate(LIBRARY subsys__mgmt__mcumgr__mgmt LOCATION EXTFLASH_TEXT NOCOPY)
    zephyr_code_relocate(LIBRARY subsys__mgmt__mcumgr__mgmt LOCATION RAM_DATA)
 
+Building the project
+********************
+
+Only specific requiremet is using childimage instead of sysbuild as xip qspi sppport is not ported to sysbuild yet.
```

Comment:
```suggestion
You must use child image when building your project, as XIP QSPI does not currently support sysbuild.
```
I'm wondering if this information should be in this section, or rather if the title of the section really covers what comes after. What do you think @kittydepa ?

Reply:
Hm, I think it is okay here since it is the first section on "building" (?) I am fine with either though.

Reply:
Let's keep as is then :)